Source/Icebreaker/Content/styles.css

```diff
@@ -189,8 +188,8 @@ body {
     color: #FFFFFF;
     cursor: pointer;
     height: 32px;
-    min-width: 96px;
-    margin: 90px 0px 0px 478px;
+    width: 96px;
+    margin: 90px 0px 0px 346px;
```

Comment:
This is sketchy-- why does this have a 346px left margin? It looks like you might be using margin to position the button all the way to the right. Don't do that. Use something like flex layout instead.